packages/manager/modules/pci/src/projects/project/data-processing/submit-job/spark-config/translations/Messages_fr_FR.json

```
@@ -0,0 +1,18 @@
+{
+  "data_processing_submit_job_stepper_spark_job_name_help": "Minimum 3 caractères",
+  "data_processing_submit_job_stepper_spark_job_name_label": "Nom du job",
+  "data_processing_submit_job_stepper_spark_swift_label": "Conteneur Swift",
+  "data_processing_submit_job_stepper_spark_swift_placeholder": "Sélectionnez un conteneur",
+  "data_processing_submit_job_stepper_spark_jobtype_label": "Type de job",
+  "data_processing_submit_job_stepper_spark_jobtype_placeholder": "Sélectionnez un type de job",
+  "data_processing_submit_job_stepper_spark_jar_file_label": "Fichier JAR",
+  "data_processing_submit_job_stepper_spark_python_file_label": "Fichier Pyhon",
+  "data_processing_submit_job_stepper_spark_main_class_label": "Classe principale",
+  "data_processing_submit_job_stepper_spark_main_class_help": "ex. org.apache.spark.examples.SparkPi",
+  "data_processing_submit_job_stepper_spark_jar_file_help": "Chemin vers votre fichier JAR au sein de votre conteneur d'objets",
+  "data_processing_submit_job_stepper_spark_version_label": "Version de Spark",
+  "data_processing_submit_job_stepper_spark_arguments_label": "Arguments (optionnel)",
+  "data_processing_submit_job_stepper_spark_arguments_placeholder": "Pressez la touche <Entrer> pour ajouter un argument",
```

```suggestion
  "data_processing_submit_job_stepper_spark_arguments_placeholder": "Pressez la touche <kbd>Entrer</kbd> pour ajouter un argument",
```
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd